evergreen.yml

```diff
@@ -235,9 +339,12 @@ tasks:
     depends_on:
       - name: build
     commands:
+      - func: "install rust toolchain"
```

Comment:
I think that you'll need to use a task group to make sure that we clean-up the registry if we did setup the driver.
My understanding is that if one of the `func` fails, the task stops. This means that the driver will still be installed because we won't clean-up.
Probably not a super big issue but it seems better to make that we return the host in a clean state.

Reply:
Okay, I switched it to a task group. 👍